sudoku/board.py

```diff
@@ -77,3 +77,13 @@ def print_board(self, plain=False) -> None:
                 print('┠───┼───┼───╂───┼───┼───╂───┼───┼───┨')
             
         print('┗━━━┷━━━┷━━━┻━━━┷━━━┷━━━┻━━━┷━━━┷━━━┛')
+
+    def save_board(self, plain=False) -> None:
+        f = open("./new_sudoku", "w")
+
+        for row in self.cells:
+            for cell in row:
+                f.write(f'{cell.value} ')                
+            f.write('\n')
+    
+        f.close()
```

Comment:
You have a point and I can see how it could be used with option `--plain`. However, it wasn't compatible with the `--generate` option at the time. 
And I liked that saving the board inside the program doesn't bring the user the inconvenience of not seeing the generated board in more human-readable form, as it does when printing just plain board.

Reply:
Since the `--plain` option not working with `--generate` has been fixed in the meantime, I think this change is unnecessary.

I can't see any inconvenience of not seeing the generated board when saving to the file. To me, it would be quite confusing to have the board automatically saved to the file and printed to the stdout at the same time. Also there is no way to specify file path/name. As I suggested, if this feature is necessary for some reason I would implement it in the way I described in the comment below. Consider also adding for example `-o\--output` option in order to be able to specify the file location/name.